sysdeps/aarch64/Versions

```diff
@@ -0,0 +1,20 @@
+libdfp {
+  LIBDFP_1.0.15 {
+    __bid_extendsdtf;
+    __bid_extendddtf;
+    __bid_trunctdtf;
+  
+    __bid_trunctfsd;
+    __bid_trunctfdd;
+    __bid_extendtftd;
+  }
+
+  LIBDFP_1.0.0 {
```

Comment:
I think these XFmode functions should be removed from aarch64. I am surprised this didn't cause the ABI test to fail.

Reply:
Hmmm so I removed this second chunk, and re-ran 'make check' and I still get "172 PASS" and no error.
Then I re-added this chunk (so back to what this patch contains), still get "172 PASS".
Then I removed the 1st chunk, 'make check' still passed with "172 PASS".
Do I need to run 'make clean' or similar before running 'make check' ?
(ie. I somehow tried to deliberately break the checks and they still passed :-) )

Reply:
The version file gets merged into a version map file (libdfp.map) used by the linker.

ld doesn't seem to care if a symbol is listed in a version file, but not present in the link unit. The ABI check only inspects the symbols exported by shared library. Thus, these are quietly ignored.

Anyhow, it's best to remove these since they should not be provided by the aarch64 version at this time (likely never).